projects/components/src/panel/header/title/panel-title.component.scss

```diff
@@ -1,20 +1,37 @@
-@import 'font';
+@import 'mixins';
 
 .ht-panel-title {
   @include subtitle-2;
-  display: inline-flex;
+  display: flex;
   align-items: center;
   border-radius: 8px;
   width: 100%;
   padding: 8px 12px;
+  justify-content: space-between;
+
+  &:hover {
+    background: $gray-1;
+    cursor: pointer;
+  }
+
+  .left-section {
+    display: flex;
+    gap: 8px;
+    flex: 1;
+  }
 
   .label {
-    margin-left: 10px;
```

Comment:
wouldn't this break anything?

Reply:
Added a gap property instead of margin left.

Reply:
Ohhk, you added gap.